client/src/components/BlogItem/BlogItem.js

```diff
@@ -0,0 +1,38 @@
+import { Col, Row, Typography } from 'antd';
+import React from 'react';
+import { Link, useRouteMatch } from 'react-router-dom';
+import { Trans } from '@lingui/macro';
+
+const { Paragraph, Text, Title } = Typography;
+
+const BlogItem = ({ postDetails }) => {
+  const { url } = useRouteMatch();
+  const { image, title, created, text, slug } = postDetails;
+  const authorFullName = `${postDetails.author_first_name} ${postDetails.author_last_name}`;
+  const temporaryDiv = document.createElement('div');
+  temporaryDiv.innerHTML = text;
+  const firstParagraph = temporaryDiv.querySelector('p').innerText;
+  return (
+    <Row className="blog-item" gutter={{ md: 24, lg: 32 }}>
+      <Col className="blog-image" sm={24} md={12} lg={10}>
+        <img src={image} alt={title} />
+      </Col>
+      <Col className="blog-meta" sm={24} md={12} lg={14}>
+        <Title level={3} underline>
+          {title}
+        </Title>
+        <Text type="secondary">
+          <Trans>
+            Published by {authorFullName} • {new Date(created).toLocaleDateString()}
+          </Trans>
+        </Text>
+        <Paragraph>{firstParagraph.slice(0, 200)}...</Paragraph>
```

Comment:
This was one of my fist approach and this will work ok if the article begins with paragraphs, but if the article will begin with a heading or the paragraph will contain other tags (ex. `strong`, `a` )  the result will be like this:
![image](https://user-images.githubusercontent.com/22684808/109420420-02667580-79db-11eb-9228-18f0ddf88e0e.png)

I will make the modifications asap if this output is acceptable.

Reply:
I'm not sure how acceptable that is, you're right. You can also use a simple regex to extract the first paragraph (just as an alternative, if you want to).